### PR TITLE
added some missing info to prod.

### PR DESCRIPTION
**Pull request by foxsan48**
I ran through this three times picking up errors in documentation and now, from start to finish it will work, with these small tweaks.

### CODE REVIEW

File: Running-Mastodon/Production-guide.md [hunk 1]
```diff
@@ -111,7 +111,12 @@ server {
 
 ## Running in production without Docker
 
-It is recommended to create a special user for mastodon on the server (you could call the user `mastodon`), though remember to disable outside login for it. You should only be able to get into that user through `sudo -u mastodon`.
+It is recommended to create a special user for mastodon on the server (you could call the user `mastodon`), though remember to disable outside login for it. You should only be able to get into that user through `sudo su mastodon`.
```
Comment (KellerFuchs) [inline Running-Mastodon/Production-guide.md]: Make it `sudo -u mastodon -s` rather than `sudo su`.

File: Running-Mastodon/Production-guide.md [hunk 2]
```diff
@@ -111,7 +111,12 @@ server {
...
+
+This command will create the user as needed:
+
+    sudo adduser --disabled-password --gecos "" mastodon
```
Comment (KellerFuchs) [inline Running-Mastodon/Production-guide.md]: Perhaps add:
- `--system` to get a system user (different UID range);
- `--group` to get a `mastodon` group as primary GID for that user;
- a `--homedir` parameter, rather than making the user's home under `/home`;
- use `--disabled-login` rather than `--disabled-password` (with the latter, one can still login with SSH keys and such).

File: Running-Mastodon/Production-guide.md [hunk 3]
```diff
@@ -207,13 +212,18 @@ It is recommended to use rbenv (exclusively from the `mastodon` user) to install
...
+rbenv install 2.4.1
+rbenv global 2.4.1
+```
 
 ## Git
 
 You need the `git-core` package installed on your system. If it is so, run the shell from the `mastodon` user:
 
-    sudo -su mastodon
+    sudo su mastodon
```
Comment (KellerFuchs) [inline Running-Mastodon/Production-guide.md]: Again, `sudo -u mastodon -s`.

File: Running-Mastodon/Production-guide.md (hunk 1 above)
Comment (foxsan48) [inline Running-Mastodon/Production-guide.md]: I have my reasons for `sudo su`, if I am wrong in this reasoning, but path seems correct and home is set correctly with `sudo su`

images showing env output differences.
### sudo -u mastodon -s
screen shot 2017-06-17 at 12 45 31 pm
### sudo su mastodon
screen shot 2017-06-17 at 12 46 29 pm

I will gladly accept being wrong, but from these outcomes I believe having the correct home is a good idea as mastodon user parts of the tutorial refer to `~/.rbenv` and etc

File: Running-Mastodon/Production-guide.md (hunk 2 above)
Comment (foxsan48) [inline Running-Mastodon/Production-guide.md]: using that requires changing home env var even when `--home dir` is used as `sudo -u <user> -s` copies the home var from the user who runs that.

Comment (foxsan48) [inline Running-Mastodon/Production-guide.md]: Okay I got it fixed! `sudo -Hsu` is needed
screen shot 2017-06-17 at 2 04 24 pm

File: Running-Mastodon/Production-guide.md [hunk 4]
```diff
@@ -111,7 +111,14 @@ server {
 
 ## Running in production without Docker
 
-It is recommended to create a special user for mastodon on the server (you could call the user `mastodon`), though remember to disable outside login for it. You should only be able to get into that user through `sudo -u mastodon`.
+It is recommended to create a special user for mastodon on the server (you could call the user `mastodon`), though remember to disable outside login for it. You should only be able to get into that user through `sudo -Hsu mastodon`.
```
Comment (wxcafe) [inline Running-Mastodon/Production-guide.md]: I'd prefer we use `sudo -iu mastodon` or `sudo su - mastodon` which are equivalent but more secure and clearer than `sudo -Hsu mastodon` imo.

File: Running-Mastodon/Production-guide.md [hunk 5]
```diff
@@ -111,7 +111,14 @@ server {
...
+
+This command will create the user as needed:
+
+    sudo adduser --system --group --disabled-login --gecos "" --home /home/mastodon mastodon
```
Comment (wxcafe) [inline Running-Mastodon/Production-guide.md]: `adduser` isn't a unix standard, it's only available on debian-based distributions. To make this more portable, we could use `useradd` which is standard. The equivalent useradd command is `sudo useradd --system --user-group --shell /bin/false --create-home --home /home/mastodon mastodon`

File: Running-Mastodon/Production-guide.md [hunk 6]
```diff
@@ -207,13 +214,18 @@ It is recommended to use rbenv (exclusively from the `mastodon` user) to install
...
+rbenv install 2.4.1
+rbenv global 2.4.1
+```
 
 ## Git
 
 You need the `git-core` package installed on your system. If it is so, run the shell from the `mastodon` user:
 
-    sudo -su mastodon
+    sudo -Hsu mastodon
```
Comment (wxcafe) [inline Running-Mastodon/Production-guide.md]: same comment as above

File: Running-Mastodon/Production-guide.md (hunk 5 above)
Comment (foxsan48) [inline Running-Mastodon/Production-guide.md]: I was going to study useradd and do that as another PR but thank you for that

File: Running-Mastodon/Production-guide.md (hunk 4 above)
Comment (foxsan48) [inline Running-Mastodon/Production-guide.md]: I prefer `sudo su` but decided against it due to previous feedback

just discovered `sudo -iu` thanks to you and damn that's exactly what I wanted! thank you

Comment (wxcafe) [inline Running-Mastodon/Production-guide.md]: huh what previous feedback? I can't see any problems with `sudo su - user`
